README.md

```diff
@@ -12,11 +14,15 @@ For more information please read the [documentation][docs].
 
 ## Installation
 
+To install Still you add it to your dependency list. You should be able to
+add it to any mix project.
+
 ### For new projects
 
-`mix archive.install hex still` to install it on your system. You only need to do this once.
+Run `mix archive.install hex still` to install the archive on your system.
```

Comment:
It's not `archive.install`, for now, is it?

Reply:
It is, but not from hex. I'll update after I merge the other PR.

Reply:
This is now correct, right? We can archive install it.

Reply:
No, you can archive install it from github, not from hex.